knowledge-graph/src/main/scala/io/renku/knowledgegraph/users/projects/finder/ProjectsFinder.scala

```diff
@@ -51,7 +51,7 @@ private class ProjectsFinderImpl[F[_]: MonadThrow](
     (findProjectsInTS(criteria) -> findProjectsInGL(criteria))
       .mapN(mergeFavouringActivated)
       .map(filterBy(criteria.filters.state))
-      .map(_.sortBy(_.name))
+      .map(_.sortBy(_.name.value.toLowerCase))
```

Comment:
We could also define the `Ordering` for this `name` type? (since we have specific types)

Reply:
Yes, good idea.